iTerm/README.md

```diff
@@ -1,28 +1,28 @@
 # iTerm2
-[iTerm2](http://www.iterm2.com/) is an open source replacement for Apple's Terminal. It's highly customizable and comes with a lot of useful features.
+> [iTerm2](http://www.iterm2.com/) is a replacement for Terminal and the successor to iTerm. It works on Macs with macOS 10.10 or newer. iTerm2 brings the terminal into the modern age with features you never knew you always wanted.
 
 ## Installation
->**Note**: Instead of downloading iTerm from the website you can use [Homebrew](http://sourabhbajaj.com/mac-setup/Homebrew/) `brew cask install iterm2`
+You can get the app from [iTerm2 downloads page](http://www.iterm2.com/downloads.html). Once downloaded, drag and drop the **iTerm** application file into your **Applications** folder.
 
-You can download the app from [here](http://www.iterm2.com/). Once downloaded, drag and drop the **iTerm** application file into your **Applications** folder.
+>**Note**: Instead of downloading and installing iTerm2 manually, you can use [Homebrew](http://sourabhbajaj.com/mac-setup/Homebrew/) `brew cask install iterm2`
 
 ## Customization
 ### Colors and Font Settings
 Here are some suggested settings you can change or set, **they are all optional**.
 
-- Set hotkey to open and close the terminal to `command + option + i`
-- Go to profiles -> Default -> Terminal -> Check silence bell to disable the terminal session from making any sound
-- Download [one of iTerm2 color schemes](https://github.com/mbadolato/iTerm2-Color-Schemes/tree/master/schemes) from here. And then set these to your default profile colors.
-- Change the cursor text and cursor color to yellow make it more visible
-- Change the font to 14pt Source Code Pro Lite. Source Code Pro can be downloaded from [here](https://github.com/adobe-fonts/source-code-pro/releases/latest).
-- If you're using BASH instead of ZSH you could add `export CLICOLOR=1` line to your `~/.bash_profile` file for nice coloring of listings
+- set hot-key to open and close the terminal to `command + option + i`
+- go to profiles -> Default -> Terminal -> Check silence bell to disable the terminal session from making any sound
+- download [one of iTerm2 color schemes](https://github.com/mbadolato/iTerm2-Color-Schemes/tree/master/schemes) and then set these to your default profile colors
+- change the cursor text and cursor color to yellow make it more visible
+- change the font to 14pt Source Code Pro Lite. Source Code Pro can be downloaded from [project's github repository](https://github.com/adobe-fonts/source-code-pro/releases/latest).
+- if you're using BASH instead of ZSH you can add `export CLICOLOR=1` line to your `~/.bash_profile` file for nice coloring of listings
```

Comment:
Seeing these changes makes me think, how nice would it be to have a style guide for writing? With details for these things (capitalized vs. un-capitalized bullet points). Then we could decide once and stick to that. Because personally I'd keep them capitalized (not saying it's better, just saying it's my way of writing).

So if we could write a style guide (or even automate it?) we would just have to decide _once_ and then be done with it.

Thoughts? @Kyslik @sb2nov @hugovk

Reply:
Personally I use [Markdown Lint](https://github.com/DavidAnson/vscode-markdownlint) but I do not apply any changes while working on this repo.